pkg/rancher-desktop/config/__tests__/settings.spec.ts

```diff
@@ -552,7 +552,6 @@ describe('settings', () => {
           rancherMode:     true,
           suppressSudo:    true,
           containerEngine: 'moby',
-          hostResolver:    true,
```

Comment:
This is the _historic_ settings, so it should stay. We should have a migration to delete it, and the _new_ settings should have it missing.